seeds/data/projects.json

```diff
@@ -513,7 +513,7 @@
     "status": "active",
     "schemaVersion": 1,
     "issueDate": "2019-08-10",
-    "expiryDate": "2023-04-23",
+    "expiryDate": "2026-04-23",
```

Comment:
Would it harm if you put a more distant year here?

Reply:
It is a good point. 

For now I have created a tech-debt ticket https://collaboration.homeoffice.gov.uk/jira/browse/ASL-4192 as it would be better if the test data dates didn't expire, and all dates could be reviewed / updated as part of that ticket.